### PR TITLE
chore: add extra information around the project

### PR DESCRIPTION
**Pull request by Jefiozie**
This PR is for better info about the project.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "vscode-xlayers",
+  "name": "xlayers-vscode",
```
Comment (Shikanime) [inline package.json]: We should put it under the xLayers organisation scope `@xlayers/vscode-extension`.

Comment (Jefiozie) [reply package.json]: I likt it 😄

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,234 @@
...
+### Scope
+
+The scope should be the name of the module affected as perceived by the person reading changelog generated from the commit messages.
+
+The following is the list of supported scopes (there might be more added in the fututre):
+
+- **core**
+- **webview**
+- **codegen**
+- **parser**
```
Comment (manekinekko) [inline CONTRIBUTING.md]: can you add `vscode` and `vscode-plugin`. wdyt?

Comment (Jefiozie) [reply CONTRIBUTING.md]: Yes, great idea!

**Pull request merged**